server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java

```diff
@@ -150,7 +151,7 @@ public static TSStatus tryCatchNonQueryException(Exception e) {
       // ignore logging sg not ready exception
       if (!(rootCause instanceof StorageGroupNotReadyException)) {
         if (((IoTDBException) e).isUserException()) {
-          LOGGER.warn(message + e.getMessage());
+          LOGGER.warn(message, e);
         } else {
           LOGGER.warn(message, e);
         }
```

Comment:
Ok, and then we don't need the if else here?

Reply:
Good catch! I removed the `if-else` statements.